This patch fixes the issue with gcc complainig about following:
error: no matching function for call to ‘max(int, long int)’

Ref: https://github.com/pytorch/pytorch/issues/70392#issuecomment-1003488196

diff --git a/src/client/linux/handler/exception_handler.cc b/src/client/linux/handler/exception_handler.cc
index b895f6d7..dc70527b 100644
--- a/src/client/linux/handler/exception_handler.cc
+++ b/src/client/linux/handler/exception_handler.cc
@@ -138,7 +138,7 @@ void InstallAlternateStackLocked() {
   // SIGSTKSZ may be too small to prevent the signal handlers from overrunning
   // the alternative stack. Ensure that the size of the alternative stack is
   // large enough.
-  static const unsigned kSigStackSize = std::max(16384, SIGSTKSZ);
+  static const unsigned kSigStackSize = std::max(16384L, SIGSTKSZ);

   // Only set an alternative stack if there isn't already one, or if the current
   // one is too small.
